packages/fluentui/react-northstar/src/components/Chat/ChatMessage.tsx

```diff
@@ -254,14 +252,16 @@ export const ChatMessage = (React.forwardRef<HTMLDivElement, ChatMessageProps>((
   const { setStart, setEnd } = useTelemetry(ChatMessage.displayName, context.telemetry);
   setStart();
 
-  const parentAttached = useContextSelector(ChatItemContext, v => v.attached);
+  const chatItemProps = useContextSelector(ChatItemContext, v => v);
```

Comment:
```suggestion
  const parentAttached = useContextSelector(ChatItemContext, v => v.attached);
  const layout = useContextSelector(ChatItemContext, v => v.unstable_layout);
```

`useContextSelector()` works in the same way as [reselect](https://github.com/reduxjs/reselect). Using `chatItemProps` as currently will cause useless re-renders.